db.migration/changelog-1.0.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+
+    <changeSet id="1" author="T1">
```

Comment:
author is usually your username or email or something like this

Reply:
Also for id it is better to use something that doesn't depend from the previous change set (e.g. current date and time), because if you have 2 branches and just increment ids, it could happen that 2 different changesets would have the same id